src/command/dev/next/router/config/parser.rs

```diff
@@ -84,7 +84,7 @@ impl<'a> RouterConfigParser<'a> {
             .and_then(|addr_and_port| Some(Uri::from_str(addr_and_port)))
             // See https://www.apollographql.com/docs/graphos/routing/self-hosted/health-checks for
             // defaults
-            .unwrap_or(Uri::from_str("127.0.0.1:8088"))
+            .unwrap_or(Uri::from_str("http://127.0.0.1:8088"))
```

Comment:
URIs need a scheme in order to be correct, so added this in for the default case